tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1324,26 +1324,32 @@ def decode_image(contents, channels=None, name=None):
       raise ValueError('channels must be in (None, 0, 1, 3)')
     substr = string_ops.substr(contents, 0, 4)
 
+    def _bmp():
```

Comment:
You should update the documentation for this function to indicate it also supports BMP in the appropriate places